extensions/blocks/eventbrite/eventbrite.php

```diff
@@ -0,0 +1,147 @@
+);
+
+/**
+ * Eventbrite block registration/dependency delclaration.
+ *
+ * @param array $attr    Eventbrite block attributes.
+ *
+ * @return string
+ */
+function jetpack_eventbrite_block_load_assets( $attr ) {
+	if ( empty( $attr['eventId'] ) ) {
+		return '';
+	}
+
+	wp_enqueue_script( 'eventbrite-widget', 'https://www.eventbrite.com/static/widgets/eb_widgets.js', array(), JETPACK__VERSION, true );
+
+	// Show the embedded version.
+	if ( empty( $attr['useModal'] ) ) {
+		wp_add_inline_script(
+			'eventbrite-widget',
+			"window.EBWidgets.createWidget({
+				widgetType: 'checkout',
+				eventId: ${attr['eventId']},
+				iframeContainerId: 'eventbrite-widget-container-${attr['eventId']}',
+			});"
+		);
+
+		return <<<EOT
+<div id="eventbrite-widget-container-${attr['eventId']}"></div>
+<noscript>
+	<a href="https://www.eventbrite.com/e/${attr['eventId']}" rel="noopener noreferrer" target="_blank">Buy Tickets on Eventbrite</a>
+</noscript>
+EOT;
+	}
+
+	// Show the modal version.
+	wp_add_inline_script(
+		'eventbrite-widget',
+		"window.EBWidgets.createWidget({
+			widgetType: 'checkout',
+			eventId: ${attr['eventId']},
+			modal: true,
+			modalTriggerElementId: 'eventbrite-widget-modal-trigger-${attr['eventId']}',
+		});"
+	);
+
+	return <<<EOT
+	<noscript><a href="https://www.eventbrite.com.au/e/${attr['eventId']}" rel="noopener noreferrer" target="_blank"></noscript>
+	<button id="eventbrite-widget-modal-trigger-${attr['eventId']}" type="button">Buy Tickets</button>
+	<noscript></a>Buy Tickets on Eventbrite</noscript>
+EOT;
+}
+
+
+/**
+ * Return fake embed data.
+ *
+ * @param array $matches Foo.
+ * @return string
+ */
+function jetpack_eventbrite_embed_handler( $matches ) {
+	if ( is_ssl() ) {
+		// phpcs:disable WordPress.WP.EnqueuedResources
+		return <<<SSLEMBED
+			<script src="https://www.eventbrite.com/static/widgets/eb_widgets.js"></script>
+			<script>
+			window.EBWidgets.createWidget({
+					widgetType: 'checkout',
+					eventId: ${matches['eventId']},
+					iframeContainerId: 'eventbrite-widget-container-${matches['eventId']}',
+				});
+			</script>
+			<div id="eventbrite-widget-container-${matches['eventId']}"></div>
+SSLEMBED;
+		// phpcs:enable
+	}
+
+	return <<<SSLINFO
+		SSL REQUIRED, Y'ALL
+SSLINFO;
+}
+wp_embed_register_handler( 'jetpack-eventbrite', '/^https?:\/\/www\.eventbrite\.com(.[a-z]+)*\/e\/.*?(?P<eventId>\d+)\/?$/', 'jetpack_eventbrite_embed_handler' );
+
+/**
+ * Handle a failing oEmbed proxy request to try embedding as a shortcode.
+ *
+ * @see https://core.trac.wordpress.org/ticket/45447
+ *
+ * @param  WP_HTTP_Response|WP_Error $response The REST Request response.
+ * @param  WP_REST_Server            $handler  ResponseHandler instance (usually WP_REST_Server).
+ * @param  WP_REST_Request           $request  Request used to generate the response.
+ * @return WP_HTTP_Response|object|WP_Error    The REST Request response.
+ */
+function jetpack_filter_oembed_result( $response, $handler, $request ) {
```

Comment:
This is copied from the Gutenberg plugin, because embeds fail when the plugin isn't enabled, as Core currently only allows proper oEmbeds to use this REST API endpoint.

I've left it in for now, (even though the PR doesn't run without the plugin), so I don't forget and wonder why everything doesn't work sometime later. 🙃